edb/lib/schema.edgeql

```diff
@@ -35,6 +35,10 @@ CREATE SCALAR TYPE schema::operator_kind_t EXTENDING std::str {
     CREATE CONSTRAINT std::one_of ('INFIX', 'POSTFIX', 'PREFIX', 'TERNARY');
 };
 
+CREATE SCALAR TYPE schema::volatility_t EXTeNDING std::str {
```

Comment:
```suggestion
CREATE SCALAR TYPE schema::volatility_t EXTENDING std::str {
```

Reply:
Why a constraint? Didn't we want to have an enum for this as there's actually an ordering that makes sense for volatility values?

Reply:
At the moment enums proper don't work nicely with assignment casts, defaults, etc. Fixing that is a separate orthogonal issue, once enums are usable in the schema we can make a PR that updates the current usage of the constraint to an actual enum type (for volatility and for other schema enums possibly). 

I am looking into fixing the issues with enums right now.